viskores/internal/Configure.h.in

```diff
@@ -327,115 +327,109 @@
 #define VISKORES_CUDA_VERSION_MINOR @VISKORES_CUDA_VERSION_MINOR@
 #endif
 
-#if __cplusplus >= 201402L || \
-    ( defined(VISKORES_MSVC) && _MSC_VER >= 1910 ) || \
-    ( defined(VISKORES_ICC)  && defined(__INTEL_CXX11_MODE__) )
```

Comment:
I guess that this had to go since it would be intel oneapi icx now and we do not have CI to test it anyways.

Reply:
The oneAPI 2025 compiler correctly sets `__cplusplus` to report that it supports C++17. Unless we want to support an earlier Intel compiler that supports enough of C++17, there should be no need for this line.